saleor/tests/e2e/checkout/taxes/test_checkout_calculate_simple_taxes_based_on_shipping_country.py

```diff
@@ -150,8 +68,10 @@ def test_checkout_calculate_simple_tax_based_on_shipping_country_CORE_2001(
         channel_id,
         variant_price,
     )
+    tax_class = {"taxClass": shipping_tax_class_id}
+    update_product(e2e_staff_api_client, product_id, tax_class)
```

Comment:
In this case, only the default country rates for flat rates should be set.

The product may not have any tax class assigned and then the saleor takes the tax class from the product type if none is assigned, it takes the default country rate. 

I think we should set tax classes only if we pass shipping_tax_rate,  product_tax_rate, product_type_tax_rate to prepare_shop